smart-common/src/main/java/org/smartdata/conf/SmartConf.java

```diff
@@ -95,6 +95,12 @@ public Map<String, String> asMap(Predicate<String> keyFilter) {
         ));
   }
 
+  public String getNonNull(String key) {
+    return Optional.of(get(key))
```

Comment:
I think it should be here `Optional.ofNullable`

Reply:
Definitely. Fixed